package.json

```diff
@@ -66,6 +67,7 @@
   },
   "dependencies": {
     "chokidar": "^3.5.3",
+    "etag": "^1.8.1",
```

Comment:
I replaced the custom implementation you were using with this module as it natively supports buffers, that we now need to support.

Reply:
`hash.update` accepts Buffer type. So I don't think this change is not necessary. Would you use the original function to make it easier to review?
https://nodejs.org/api/crypto.html#hashupdatedata-inputencoding